lib/config/migration.ts

```diff
@@ -4,6 +4,7 @@ import { dequal } from 'dequal';
 import { logger } from '../logger';
 import { clone } from '../util/clone';
 import { regEx } from '../util/regex';
+import { fixShortHours } from '../workers/branch/schedule';
```

Comment:
Nope. Config can't have a dependency on workers.

So this function needs to be moved to config or utils.

Reply:
Moved it to `util/url.ts`